website/www/site/content/en/documentation/sdks/yaml-udf.md

```diff
@@ -133,11 +184,26 @@ expressions are supported as well
       another_col: "col2 + col3"
 ```
 
+keeping in mind that any fields not intended to be included in the output should be added to the `drop` field.
+
+If one wanted to select a field that collides with a[reserved SQL keyword](https://calcite.apache.org/docs/reference.html#keywords), the field(s) must be surrounded in backticks. For example, say the incoming PCollection has a field "timestamp", one would have to write:
```

Comment:
```suggestion
If one wanted to select a field that collides with a [reserved SQL keyword](https://calcite.apache.org/docs/reference.html#keywords), the field(s) must be surrounded in backticks. For example, say the incoming PCollection has a field "timestamp", one would have to write:
```

I'm just going to apply this so that it gets in before the cut